### PR TITLE
Make some changes as we change our default branch from 'master' to 'main'.

### PR DESCRIPTION
**Pull request by tmds**
- Updates urls from 'master' to 'main'.
- Remove old templates, image streams and install scripts.
- Clean-up the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
-Versions
-----------------
+The images support OpenShift [source-to-image](https://github.com/openshift/source-to-image).
+
+## Supported Versions
 
-.NET Core versions currently provided are:
+| Version | OS | Documentation
+|--|--|--|
+| 6.0 | UBI 8, Fedora | [SDK image](6.0/build/README.md) <br/> [Runtime image](6.0/runtime/README.md)
```
Comment (omajid) [inline README.md:11]: Is the Fedora image maintained/tested?

File: README.md [hunk 2]
```diff
@@ -30,149 +32,22 @@ $ git clone https://github.com/redhat-developer/s2i-dotnetcore.git
...
-### Windows
-
-script: https://raw.githubusercontent.com/redhat-developer/s2i-dotnetcore/master/install-imagestreams.ps1
-
-For example, to install the `rhel` based images and add a secret for authenticating against the `registry.redhat.io` registry:
-
-```sh
-PS > Invoke-WebRequest https://raw.githubusercontent.com/redhat-developer/s2i-dotnetcore/master/install-imagestreams.ps1 -UseBasicParsing -OutFile install-imagestreams.ps1
-PS > .\install-imagestreams.ps1 -OS rhel -User <subscription_username> -Password <subscription_password>
+oc apply -f https://raw.githubusercontent.com/redhat-developer/s2i-dotnetcore/main/dotnet_imagestreams.json
```
Comment (omajid) [inline README.md:43]: :tada:

File: README.md (hunk 1 above)
Comment (tmds) [reply README.md:11]: No, it's available as-is for the community to contribute to.

File: dotnet_imagestreams.json [hunk 3]
```diff
@@ -25,7 +25,7 @@
                         "name": "latest",
```
[the remark this answers is not in the document]
Comment (tmds) [reply dotnet_imagestreams.json:25]: Yes, all the files will have the same content then.

**Comment by tmds** [comment]
Thanks for the review, @omajid.

**Pull request merged**